.circleci/pyscripts/generate_main.py

```diff
@@ -26,13 +27,13 @@ def combine_workflow_path(job_entry_spaces, workflow_path):
     _entries = []
     combined_entries = []
     for index, element in enumerate(workflow_path):
-        re_match = re.match("^"+(" "*job_entry_spaces)+"- ", element)
+        re_match = re.match("^" + (" " * job_entry_spaces) + "- ", element)
```

Comment:
You might want to consider using Python f-strings to make this cleaner to read:
https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498 

Not absolutely required, but just a note.